packages/bot-web-ui/src/components/run-panel/run-panel.scss

```diff
@@ -112,7 +112,7 @@
     }
     &-tab {
         &__content {
-            height: 52rem;
+            height: 50rem;
```

Comment:
we should put flexible height such as it works fine with height: calc(100vh - 42rem);
lets try doing this @farabi-deriv

Reply:
this has been resolved in a separate PR